spyder/plugins/configdialog.py

```diff
@@ -398,7 +401,13 @@ def save_to_conf(self):
         for radiobutton, (option, _default) in list(self.radiobuttons.items()):
             self.set_option(option, radiobutton.isChecked())
         for lineedit, (option, _default) in list(self.lineedits.items()):
-            self.set_option(option, to_text_string(lineedit.text()))
+            option_value = lineedit.text()
+            old_value = self.get_option(option, _default)
+            if isinstance(old_value, list):
+                option_value = [int(e) for e in option_value.split(',')]

```

Comment:
To address this I think we we should change it to

```python
try:
    option_value = [int(e) for e in option_value.split(',')]
except ValueError:
    option_value = [e for e in option_value.split(',')]
```

@goanpeca, what do you think?